### PR TITLE
Update rustc to 1.28.0

### PR DESCRIPTION
**Pull request by alatiera**

### CODE REVIEW

**Comment by alatiera** [comment]
bot, build

File: org.freedesktop.Sdk.Extension.rust-stable.json [hunk 1]
```diff
@@ -33,32 +33,32 @@
...
         },
         {
           "type": "archive",
           "only-arches": [
             "arm"
           ],
-          "url": "https://static.rust-lang.org/dist/rust-1.27.2-arm-unknown-linux-gnueabihf.tar.gz",
-          "sha256": "9948db9247f6d853a3c3392239ee08fc3ce1d2044cf282e2286403f4cbbcd799"
+          "url": "https://static.rust-lang.org/dist/rust-1.28.0-arm-unknown-linux-gnueabihf.tar.gz",
+          "sha256": "346558d14050853b87049e5e1fbfae0bf0360a2f7c57433c6985b1a879c349a2"
```
Comment (danigm) [inline org.freedesktop.Sdk.Extension.rust-stable.json]: This hash isn't correct, the correct one is: "5d93473fdaa85379ec83f5318b0316f968063198f9ef10ef2cadb5e73cfbc667"
take a look at this: https://static.rust-lang.org/dist/rust-1.28.0-arm-unknown-linux-gnueabihf.tar.gz.sha256

Comment (alatiera) [reply org.freedesktop.Sdk.Extension.rust-stable.json]: whoops, write I will fix it in a sec

**Comment by alatiera** [comment]
I already updating the rustc extension for fd.o 18.07 so I figured I would make a PR since I already had the hashes

**Pull request merged**